rs/nns/governance/BUILD.bazel

```diff
@@ -65,7 +65,6 @@ DEPENDENCIES = [
     "//rs/nns/sns-wasm",
     "//rs/protobuf",
     "//rs/registry/canister",
-    "//rs/rust_canisters/dfn_http_metrics",
```

Comment:
note to reviwer, I noticed this should have been removed previously, but not related to SNS-W